src/test/resources/features/CreateCoordinate.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Create Coordinate
+  In order to use the app
+  As a user
+  I want to create coordinate and check throw errors
+
+  Background:
+    Given There is a registered user with username "username" and password "password" and email "user@domain.com"
+
+  Scenario: Create new coordinate
+    Given I can login with username "username" and password "password"
```

Comment:
Use `I login as "username" with password "password"` instead of `I can login with username "username" and password "password"`. The current one logs in and then checks, but it is not needed in this use case as we are not testing that users are actually registered

Reply:
you are right i will make the change
I will continue to implement